sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/DoubleExponentialHistogramData.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import com.google.auto.value.AutoValue;
+import java.util.Collection;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * An exponential histogram metric point, as defined by the OpenTelemetry Exponential Histogram.
+ *
+ * <p>See:
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#exponentialhistogram
+ *
+ * <p><i>Note: This is called "DoubleExponentialHistogramData" to reflect which primitives are used
+ * to record it, however "ExponentialHistogram" is the equivalent OTLP type.</i>
+ */
+@Immutable
+@AutoValue
+public abstract class DoubleExponentialHistogramData
+    implements Data<DoubleExponentialHistogramPointData> {
+  DoubleExponentialHistogramData() {}
+
+  public static DoubleExponentialHistogramData create(
```

Comment:
this method needs some javadoc. Thanks!

Reply:
I missed that, thanks. Just pushed it up 👍